concent_api/core/message_handlers.py

```diff
@@ -795,6 +799,7 @@ def handle_send_force_payment(
         cut_off_time < subtask_results_accepted.payment_ts + settings.PAYMENT_DUE_TIME
         for subtask_results_accepted in client_message.subtask_results_accepted_list
     ):
+        logging.log_payment_time_exceeded(logger, client_message.subtask_results_accepted_list)
```

Comment:
I've move this whole `if` block into `bankster`, as a part of bugfix that's not in `master` yet, so You can move this line to line 828, and we will avoid conflicts no matter which PR will get merged sooner.

Reply:
After discuss I've added comment in you PR: https://github.com/golemfactory/concent/pull/1015#discussion_r241343589